pom.xml

```diff
@@ -8,7 +8,7 @@
 
   <artifactId>bitbucket-pullrequest-builder</artifactId>
   <name>Bitbucket Pullrequest Builder Plugin</name>
-  <version>1.4.14-SNAPSHOT</version>
+  <version>1.4.15</version>
```

Comment:
This line is using for my release script(maven release plugin), so please do not change it.